README.md

```diff
@@ -137,33 +137,43 @@ See [the demo app](demo/App.vue) for a more comprehensive example.
 
 ## Advanced Usage
 
-### Customizing Styling
+### Customizing Vuetify Configuration
+
+If your downstream application is also using Vuetify and needs to pass additional configuration
+options, it must be careful to coordinate with Girder web components.
 
-If your downstream application is also using Vuetify, and needs to pass additional configuration
-options to Vuetify, it must be careful to coordinate with Girder web components.
+Additional Vuetify configuration should inherit from Girder web components' own configuration:
 
-Additional Vuetify configuration should inherit from Girder web components own configuration:
 ```javascript
 import { merge } from 'lodash';
 import { vuetifyConfig as girderVuetifyConfig } from '@girder/components/src/utils';
 
-// More complicated merging could use Lodash's _.merge
 const appVuetifyConfig = merge(girderVuetifyConfig, {
   icons: {
-    'myCustomIcon': 'mdi-custom-icon'
+    values: {
+      myCustomIcon: 'mdi-custom-icon'
+    }
   }
 });
 ```
 
-Any custom configuration must be passed to Vuetify before Girder web components is
+> **Note:** You must use the mdi icon pack. Icon packs cannot be mixed.
+
+Any custom configuration must be passed to Vuetify **after** Girder web components is
 installed to Vue:
+
 ```javascript
-// This order is important
-Vue.use(Vuetify, appVuetifyConfig);
+import Vuetify from 'vuetify/lib';
+// Order is important: your custom config should come after Girder
```

Comment:
It looks like since the config of Vuetify is not feed through Vue.use(Vuetify, config). The order doesn't matter anymore.